dockstore-webservice/src/main/resources/migrations.1.9.0.xml

```diff
@@ -41,4 +41,14 @@
             </column>
         </addColumn>
     </changeSet>
+    <changeSet author="aduncan (generated)" id="addLegacyVersionColumn">
+        <addColumn tableName="workflowversion">
+            <column name="islegacyversion" type="bool"/>
+        </addColumn>
+        <sql dbms="postgresql">
+            UPDATE workflowversion SET islegacyversion=true;
+            UPDATE workflowversion SET islegacyversion=false WHERE id IN (SELECT wv.id FROM workflowversion wv JOIN workflow_workflowversion wwv ON wv.id=wwv.workflowversionid WHERE wwv.workflowid IN (SELECT id FROM workflow WHERE mode='DOCKSTORE_YML'));
```

Comment:
Double-check, this seems to set all versions of workflows in mode DOCKSTORE_YML to not be in legacy mode. 
Will this be true while people are migrating from one to another?

Maybe it doesn't matter if this runs before people have access to workflows via GitHub apps, but then why set anyone to false?

Reply:
This is only true before anyone has migrated. The goal is that before any migrations happen, all dockstore.yml versions are set to false and all other versions are set to true. Once people can migrate, then there is the case where a dockstore.yml workflow has both legacy and new versions.

The purpose of this field is so that for migrated workflows we know which versions are legacy and which are new, in order to distinguish them in the future (especially useful if we end up with the bot for making PRs).

Reply:
Just noticed this... This is going to have to be updated for #3257, where the workflow_workflowversion table no longer exists.

Reply:
:'(